ansible_base/oauth2_provider/views/token.py

```diff
@@ -51,7 +51,41 @@ def create_token_response(self, request):
             return request.build_absolute_uri(), {}, str(e), e.status_code
 
 
+class OAuth2TokenPermission(permissions.BasePermission):
```

Comment:
@AlanCoding thoughts on this implementation?

Reply:
Should we move any custom permissions for apps into a "permissions" folder or (or permissions.py)?

Reply:
I can move it into permissions.py, sure